packages/components/src/ui/item-group/item.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import { contextConnect, PolymorphicComponentProps } from '../context';
+import type { ItemProps } from './types';
+import { useItem } from './use-item';
+import { ItemView } from './styles';
+
+function Item( props: PolymorphicComponentProps< ItemProps, 'div' > ) {
```

Comment:
Is it missing `forwardedRef: Ref< any >` ?

Reply:
It's really weird that `contextConnect` didn't yell about it missing the ref as the first argument to `contextConnect` is a function accepting PolymorphicComponentProps and a ref 🤔